extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1781,9 +1788,18 @@ export class AmpStoryPage extends AMP.BaseElement {
           LocalizedStringId.AMP_STORY_PAGE_ATTACHMENT_OPEN_LABEL
         );
 
+      const container = this.win.document.createElement('div');
+      container.classList.add('i-amphtml-page-attachment-host');
+      container.setAttribute('role', 'button');
+
       this.mutateElement(() => {
         textEl.textContent = openLabel;
-        this.element.appendChild(this.openAttachmentEl_);
+        this.element.appendChild(container);
```

Comment:
Attachments are no longer opening when using TalkBack.
This appears to be because the click target is in the shadow dom.
It works when the click listener is on the container. I think this needs to be fixed in a follow up PR.